packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -115,6 +115,7 @@ abstract class ScrollView extends StatelessWidget {
     this.keyboardDismissBehavior = ScrollViewKeyboardDismissBehavior.manual,
     this.restorationId,
     this.clipBehavior = Clip.hardEdge,
+    this.hitTestBehavior = HitTestBehavior.opaque,
```

Comment:
I wonder if this might be better in the inherited ScrollBehavior. Every scrollable inherits the behavior that includes things like the physics and the devices that can drag scroll... 
We could even do both, like how you can set the physics on the ListView (etc), but if not provided it will fall back to the physics from the ScrollBehavior.

If we put this in the ScrollBehavior, then users can configure it for all scrollables rather than having to go set this property on all of them individually.

Reply:
The fallback sounds better. I will implement it then.

Reply:
The change of default hit test behavior on `Scrollable` has a very limited and rare use cases and I think it's better to keep it as a constructor parameter since `ScrollBehavior` encapsulates different behavioral configurations for `Scrollable` which can vary based on Platform. WDYT? @Piinks

Reply:
That sounds good to me. Thanks for considering it. We can always add it later if there is a request.